### ISSUE TITLE
[Legitimate Site Blocked] uni.live

### ISSUE THREAD

**Issue by uri-bloXroute** [MetaMask/eth-phishing-detect#14898]
hey,
bloXroute is building a new Uniswap front-end which includes:
- front-running protection
- lower front-end fee
- the killer feature we're still working on is showing price changes in real-time (by using the latest prices of MEV-boost bids)

we got flagged, I'm not sure why, and would like it to be removed.

**MOST IMPORTANTLY** after discussion with Uniswap Labs, we're going to re-launch using swap.live  and I want to make sure this domain doesn't get flagged since a lot of our users got scared by it.

If you want any additional info / confirmation pls let me know (you can also ping Austin from Uniswap, they're aware of our efforts)

Thanks!
Uri

https://uni.live/

**Comment by AlexHerman1** [comment]
hey thanks ill inquire with them

**Comment by AndrewMohawk** [comment]
This list is for phishing/scam sites and I don't believe this is what is occurring on this domain and can be removed from this blocklist

**Comment by uri-bloXroute** [comment]
thanks @AndrewMohawk 

just spoke with Connor & Austin from Uniswap, and they were also like "why would they reach out to us, it has nothing to do with us"

TLDR- if you need to confirm it's me samczsun and many others know me and can DM me to make sure it's legit
can hop on a call too (honestly not sure what you need to decide something is legit)

**Comment by samczsun** [comment]
Looks like the block was merged in https://github.com/MetaMask/eth-phishing-detect/pull/14466
